lms/djangoapps/teams/tests/test_views.py

```diff
@@ -1708,3 +1709,50 @@ def test_download_csv(self):
     def get_url(course_id):
         # This strategy allows us to test with invalid course IDs
         return reverse('team_membership_bulk_management', args=[course_id])
+
+    def test_upload_valid_csv_simple(self):
+        self.create_and_enroll_student(username='a_user')
+        csv_content = 'user,mode,topic_0' + '\n'
```

Comment:
question: where is the teams_configuration for this course defined?

Reply:
class TeamAPITestCase_setUpClass